assets/javascripts/discourse/templates/modal/query-help.hbs

```diff
@@ -0,0 +1,50 @@
+{{#d-modal-body title="explorer.help.modal_title"}}
+  <h2>{{i18n "explorer.help.auto_resolution.title"}}</h2>
```

Comment:
I wonder if the modal content should just be one translatable string instead?

Reply:
Yeah, it'll definitely make things simpler.

Reply:
@ZogStriP  I wonder if it's all right to put that much HTML in a translatable string when putting all modal content into it. The modal even contains a table:

<img width="733" alt="Screenshot 2021-04-13 at 15 06 03" src="https://user-images.githubusercontent.com/1274517/114543311-17852400-9c6a-11eb-9ed9-7e3b045005cf.png">

Reply:
I've split the modal content into 4 pieces and eventually removed the table and started using a list instead. It's a bit tricky business to use a table here, we need a CSS class to be applied to the table, so we can't put the whole table into a resource. With the list it looks like this:

<img width="695" alt="Screenshot 2021-04-15 at 17 21 47" src="https://user-images.githubusercontent.com/1274517/114876202-26590b80-9e0f-11eb-95a6-2448c38064a5.png">